semeio/jobs/rft/gendata_rft.py

```diff
@@ -0,0 +1,58 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def _write_points(points, fname):
+    with open(fname, "w+") as fh:
+        fh.write("\n".join([str(p.get_pressure()) for p in points]))
+    logger.info("Forward model script gendata_rft.py: Wrote file {}".format(fname))
+
+    with open(fname + "_active", "w+") as fh:
+        fh.write("\n".join([str(int(p.is_active())) for p in points]))
+
+
+def _write_inactive_info(points, fname, zonemap):
+    with open(fname + "_inactive_info", "w+") as fh:
+        fh.write(
+            "\n".join([p.inactive_info(zonemap) for p in points if not p.is_active()])
+        )
+
+
+def _populate_trajectory_points(
+    well, date, trajectory_points, ecl_grid, ecl_rft, zonemap
+):
+    try:
+        rft = ecl_rft.get(well, date)
+    except KeyError:
+        logger.error(
+            "Forward model script gendata_rft.py: No RFT data written for well ({}) and date ({})".format(
+                well, date
+            )
+        )
+        return []
```

Comment:
I'm not sure - but think its more clear if None is returned at this point - and by that the test on line 51 would make more sense - and could also be rewritten to `if not trajectory_points is None` or something.

Reply:
I think I disagree.

I like that a function always returns objects with the same attributes. So a function that usually returns a list of content should return an empty list if nothing happens.

The check on line 51 should be good. It's pythonic to check the emptyness of lists by using `if my_list`.

But I'm happy to be disproved :=)

Reply:
as said - i'm not sure.. i'm totally fine with the code as is - just wanted to discuss the exception handling and returning an empty list :+1: and you have a good point - it's always good to return objects with same attributes!